buildSrc/src/main/java/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -25,6 +25,7 @@
 import com.google.firebase.gradle.plugins.apiinfo.GetMetalavaJarTask;
 import com.google.firebase.gradle.plugins.ci.Coverage;
 import com.google.firebase.gradle.plugins.ci.device.FirebaseTestServer;
+import com.google.firebase.gradle.plugins.license.LicenseResolverPlugin;
```

Comment:
Is this change related to the proguard logic?

Reply:
yes and no :)

It used to live in the root project configuration alongside the proguard logic. So it makes sense to move it here